src/Routing/Filter/LocaleSelectorFilter.php

```diff
@@ -15,6 +15,7 @@
 namespace Cake\Routing\Filter;
 
 use Cake\Event\Event;
+use Cake\I18n\I18n;
```

Comment:
I think that's ok. We can always make it an optional dependency if/when we split this package out.